recipes/python-irodsclient/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "python-irodsclient" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "ca4fb5c390744420d2265043796a8fbb5f9ca1e3ac2c19e13e3072fc52161daf" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
It looks like this package might be able to be `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

My only concern is that the pypi meta info says "posix", so I'm not sure if there is an issue building/running this package on windows. If there is, that build should be skipped and you can't use `noarch`. If this is ok on windows, then consider using `noarch`.